### DISCUSSION TITLE
How can users interact with large audio files more quickly on a web page?

### DISCUSSION THREAD

**Discussion by slarkvan** [katspaugh/wavesurfer.js#3766] [Q&A]
My server's audio files are very large. My current approach on the web client is to first use ws.load(audioUrl), and then the user has to wait for the audio to completely download before they can interact. Is there a way to allow users to interact with the audio player more quickly?

**Comment by entonbiba** [comment]
@slarkvan use mediaelement to load large audio files.

```
var wavesurfer = WaveSurfer.create({
            container: '#waveform',
            waveColor: 'violet',
            progressColor: 'purple',
            backend: 'MediaElement',
            mediaType: 'audio',
            mediaControls: true
});
wavesurfer.load('example.mp3');
```

Another way you could do it is to split up the audio data into chunks and load the buffer after if you don't want to use medialement backend.
